CODEOWNERS

```diff
@@ -223,15 +218,14 @@
 # The following filepaths should be sorted so that more specific paths occur
 # after the less specific paths, otherwise the ownership for the specific paths
 # is not properly picked up in Github.
-* @cilium/tophat
```

Comment:
Did you check whether this made any difference? From what I remember, [we ignore the catch-up rule](https://github.com/cilium/cilium/blob/main/.github/workflows/lint-codeowners.yaml#L60C1-L63) (on purpose) when validating that all files are covered in CI:

```
          # Copy all patterns from CODEOWNERS, but skipping the comments
          # ('^[^#]') and the catch-all '*' rule (the only one with a single
          # character, we skip it with '^.[^ ]').
```

Reply:
oh interesting. now i'm starting to suspect if this workflow is doing what i think it should be doing. i'll do some investigation and report back 🧑‍🔬

Reply:
ok yeah lint-codeowners.yaml is ignoring the catch-all, so the commit message is incorrect. all these files must have added before lint-codeowners.yaml was introduced. i'll update it 📝

Reply:
updated the commit message 📝